content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,116 @@
+
+The separation between the L4 secure overlay layer and L7 processing layer allows incremental adoption of the ambient mode data
+plane in contrast to the earlier binary “all in” injection of sidecars. Users can start with the secure overlay layer which
+offers mTLS with cryptographic identity, simple L4 authorization policy, and telemetry. Later on, complex L7 handling such as
+retries, traffic splitting, complex load balancing, and observability collection can be enabled on a case-by-case basis.
+
+## What is in the scope of the Beta?
+
+We recommend you explore the following Beta functions of ambient mode in production with appropriate caution, after validating
+them in test environments:
+
+- [Install](/docs/ambient/install/).
+- [Adding your workloads to the mesh](/docs/ambient/usage/add-workloads/) to gain mutual TLS with cryptographic identity, [L4 authorization policies](t/docs/ambient/usage/l4-policy/), and telemetry.
+- [Configure waypoint proxies](/docs/ambient/usage/waypoint/) to [use L7 functions](/docs/ambient/usage/l7-features/) such as traffic shifting, request routing, rich authorization policy enforcement.
+- Istio ingress gateway can work with workloads in ambient mesh supporting all existing Istio APIs.
+- Use `istioctl` to operate waypoints, and troubleshoot ztunnel & waypoints.
+
+### Alpha features
+
+Other features we want to include in ambient mode have been implemented but remain in Alpha status in this release. Please help
+test them, so they can be promoted to Beta in 1.23 or later:
+
+- Multi-cluster installations
+- DNS proxying
+- Interoperability with sidecars
+- IPv6/Dual stack
+- SOCKS5 support (for outbound)
+- Istio’s classic APIs (`VirtualService` and `DestinationRule`)
+
+### Roadmap
+
+We have a number of features which are not yet implemented in ambient mode but are planned for upcoming releases:
+
+- Controlled egress traffic
+- Multi-network support
+- Improve `status` messages on resources to help troubleshoot and understand the mesh
+- VM support
+
+## What about sidecars?
+
+Sidecars are not going away, and remain first-class citizens in Istio. You can continue to use sidecars and they will remain
+fully supported.  For any feature outside of the Alpha or Beta scope for ambient mode, you should consider using the sidecar
+pattern before the feature is added to ambient mode. Some use cases, such as traffic shifting based on source labels, will
+continue to be best implemented using the sidecar pattern. While we believe most use cases will be best served with a mesh in
+ambient mode, the Istio project remains committed to ongoing sidecar mode support.
+
+## Try Istio’s new sidecar-less ambient mode today!
+
+With the 1.22 release of Istio and the beta release of ambient mode, it will be easier than ever to try out Istio on your own
+workloads. Follow the [get started guide](/docs/ambient/getting-started/) to explore ambient or [user guide](https://preliminary.istio.io/latest/docs/ambient/usage/)
```

Comment:
```suggestion
workloads. Follow the [getting started guide](/docs/ambient/getting-started/) to explore ambient or [user guide](https://preliminary.istio.io/latest/docs/ambient/usage/)
```

We'll need to update this link once the prelim docs are merged so that we link to the live site.